tests/unit/lms/views/exceptions_test.py

```diff
@@ -44,7 +44,11 @@ def test_hapi_error(self, assert_response, pyramid_request):
 
         template_data = ExceptionViews(exception, pyramid_request).hapi_error()
 
-        assert_response(template_data, 500, message="hapi_error_test_message")
+        assert_response(
+            template_data,
+            500,
+            message="ExternalRequestError(message='hapi_error_test_message', response=None)",
```

Comment:
This is because [the exception view for `HAPIError` calls `str()` on the exceptoin](https://github.com/hypothesis/lms/blob/2e5f76ab1edb8c82a0e4d24c8ba41470e1c62283/lms/views/exceptions.py#L36-L38). I don't think we want that as these `"ExternalRequestError(...)"` strings are meant for our monitoring systems not for end users. I should fix that exception view